ChemistryLib/PhreeqcIOData/CreateKineticReactant.cpp

```diff
@@ -29,17 +29,28 @@ std::vector<KineticReactant> createKineticReactants(
         //! \ogs_file_param{prj__chemical_system__kinetic_reactants__kinetic_reactant__name}
         auto name = reactant_config.getConfigParameter<std::string>("name");
 
+        auto chemical_formula_config =
+            //! \ogs_file_param{prj__chemical_system__kinetic_reactants__kinetic_reactant__chemical_formula}
+            reactant_config.getConfigParameterOptional<std::string>(
+                "chemical_formula");
```

Comment:
```c++
reactant_config.getConfigParameter<std::string>(
```
```suggestion
                "chemical_formula", "");
```

And same for the parameters below. The if-condition is already included in the `getConfigParameter` function.

Reply:
I have not noticed the default setting contained in the `getConfigParameter` ever.